pv_site_api/main.py

```diff
@@ -319,53 +320,68 @@ def get_pv_estimate_clearsky(site_uuid: str, session: Session = Depends(get_sess
     """
     ### Gets a estimate of AC production under a clear sky
     """
+    clearsky_estimates = get_pv_estimate_clearsky_many_sites(site_uuid, session)
```

Comment:
Should we add 404 logic like they have above?

https://github.com/openclimatefix/pv-site-api/pull/84/files#diff-aa1f06de99546f36c9596d3097374fac5b2c99ac6cae3d4cce042440bcfe46f6R274-R280

Reply:
yep, done!

Reply:
I always get a bit worried when we take stuff. Should this be put back in as @AndrewLester suggests